source/nodejs/adaptivecards-designer/src/adaptive-card-schema.ts

```diff
@@ -147,6 +147,7 @@ export const adaptiveCardSchema =
           "$ref": "#/definitions/Actions"
         },
```

Comment:
Add more than one CarouselPages, then click `Preview` and uncheck `Preview`, `add a page` button disappeared.
![image](https://user-images.githubusercontent.com/6290692/179060553-3ecf988a-0221-4526-a2b6-04d648f2cddc.png)

Reply:
Is the carousel selectable in this state? I believe losing state when we switch between preview and design mode is the intended behavior

Reply:
@anna-dingler - i agree. the current behavior (hiding the "add a page" on preview mode) is as expected. this is consistent with the other design/preview workflows